localstripe/resources.py

```diff
@@ -748,6 +781,14 @@ def _api_list_tax_ids(cls, id, **kwargs):
     # Delete single source by id:
     ('DELETE', '/v1/customers/{id}/sources/{source_id}',
      Customer._api_remove_source),
+
+
```

Comment:
Can you please remove this empty lines (and the same below), for consistency with the rest of the file?